benchmark/main.cpp

```diff
@@ -519,6 +527,8 @@ MBED_NOINLINE static int benchmark_aes_gcm()
 
         ret = mbedtls_gcm_setkey(&gcm, MBEDTLS_CIPHER_ID_AES, tmp, keysize);
         if (ret == MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE) {
+            /* Do not consider this as a failure */
```

Comment:
Note: Obviously this will work only for AES feature unavailable, and not in case an alternative gcm implementation returns the uncreated yet `MBEDTLS_GCM_FEATURE_UNAVAILABLE`

This is not a review comment, but an notice for future implementation